frontend/src/app/contribute.cljs

```diff
@@ -124,8 +126,25 @@
 
     (instructions-item nil "Submit")]))
 
+(defn display-error-middle-top []
+  (if @error-description
+    [:div {:class "text-center rounded"}
```

Comment:
What about using a bootstrap alerts?

```clojure
[:div {:class "alert alert-danger alert-dismissible fade show text-center"}
 [:strong @error-title]
 [:p @error-description]
 [:button {:type "button" :class "btn-close" :data-bs-dismiss "alert"}]]
```

They look pretty a and they allow you to dismiss them